supportedBackends/google/pipelines/v2beta/src/main/scala/cromwell/backend/google/pipelines/v2beta/LifeSciencesFactory.scala

```diff
@@ -154,13 +154,17 @@ case class LifeSciencesFactory(applicationName: String, authMode: GoogleAuthMode
        */
       val adjustedBootDiskSize = {
         val fromRuntimeAttributes = createPipelineParameters.runtimeAttributes.bootDiskSize
-        // Compute the decompressed size based on the information available
-        val userCommandImageSizeInBytes = createPipelineParameters.jobDescriptor.dockerSize
+
+        // Compute the decompressed size based on the information available. If we couldn't get the image size,
+        // default to 30GB. Defaulting to 0 can cause task to run out of disk. (more in AN-300)
```

Comment:
This is 30 GB extra for the image, right? Not a 30 GB disk overall.

Reply:
Correct!